packages/odyssey-react-mui/src/components/Link/Link.tsx

```diff
@@ -0,0 +1,44 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import React, { forwardRef, ReactElement } from "react";
+
+import { Link as MuiLink, SvgIcon } from "@mui/material";
+import type { LinkProps as MuiLinkProps } from "@mui/material";
+
+export interface LinkProps extends MuiLinkProps {
+  icon?: ReactElement;
+}
+
+export const Link = forwardRef<HTMLLinkElement | HTMLAnchorElement, LinkProps>(
+  (props) => {
+    const { icon, children, target } = props;
+    return (
+      <MuiLink {...props}>
+        <span className="icon">{icon}</span>
+        {children}
+        {target === "_blank" && (
+          <span className="indicator" role="presentation">
+            <SvgIcon viewBox="0 0 16 16" titleAccess="external link icon">
```

Comment:
@jeffbelser-okta @josesolano-okta any sense of how we want to handle icons with MUI?

Don't think it's blocking, but we may want to open a ticket for revisiting this when we start that work.